strategies.py

```diff
@@ -190,16 +190,23 @@ def is_done(self):
         if self.root.position.is_game_over():
             return True
 
-        if self.root.position.n >= MAX_GAME_DEPTH:
-            return True
         return False
 
     def show_path_to_root(self, node):
         pos = node.position
+        diff = node.position.n - self.root.position.n
         if len(pos.recent) == 0:
             return
-        moves = list(map(coords.to_human_coord,
-                         [move.move for move in pos.recent[self.root.position.n:]]))
+        fmt = lambda move: "{}-{}".format('b' if move.color == 1 else 'w',
+                                          coords.to_human_coord(move.move))
+        path = " ".join(list(map(fmt,
```

Comment:
this is actually more verbose than the comprehension:
`path = " ".join(fmt(move) for move in pos.recent[-diff:])`

Reply:
quite a bit more verbose actually.  Done!